Competition2017/src/competition/subsystems/feeder/SideFeederSubsystem.java

```diff
@@ -0,0 +1,186 @@
+package competition.subsystems.feeder;
+
+import org.apache.log4j.Logger;
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideFeederSubsystem extends BaseSubsystem {
+
+    private static Logger log = Logger.getLogger(SideFeederSubsystem.class);
+
+    protected final XCANTalon feederMotor;
+
+    public enum FeederSide {
```

Comment:
Can we use the same enum for shooter and feeder sides? if we do, we can quickly check that two objects are on the same side.